examples/spark2/src/main/java/org/apache/carbondata/examples/sql/CarbonSessionExample.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.carbondata.examples.sql;
+
+import org.apache.carbondata.core.constants.CarbonCommonConstants;
+import org.apache.carbondata.core.util.CarbonProperties;
+import org.apache.carbondata.examples.util.ExampleUtils;
+import org.apache.log4j.PropertyConfigurator;
+import org.apache.spark.sql.SparkSession;
+
+import java.io.File;
+import java.io.IOException;
+
+public class CarbonSessionExample {
+
+  public static void main(String[] args) {
+    File file = new File(CarbonSessionExample.class.getResource("/").getPath() + "../..");
+    try {
+      String rootPath = file.getCanonicalPath();
+      System.setProperty("path.target", rootPath + "/examples/spark2/target");
+      PropertyConfigurator.configure(rootPath + "/examples/spark2/src/main/resources/log4j.properties");
+      CarbonProperties.getInstance().addProperty(CarbonCommonConstants.ENABLE_QUERY_STATISTICS, "true");
+      SparkSession spark = ExampleUtils.createCarbonSession("JavaCarbonSessionExample", 1);
```

Comment:
I think you can change this variable name to 'carbon' to tell the user that it is not a raw SparkSession.

Besides, I think it's better to add an example for DataFrame  (write dataframe to carbon table).

Reply:
Thank you for the suggestion. I updated the variable name. And I think I will start the DataFrame example in another pr.